packages/core/src/components/badge/badge.scss

```diff
@@ -0,0 +1,28 @@
+:host {
```

Comment:
**suggestion:** as commented [here](https://github.com/juntossomosmais/atomium/pull/239/files#r1297725689), the host may not be needed, and also ionic-badge has `    display: inline-block;` so the problem that you need this css here, is fixed

Reply:
Nice too!

Made: https://github.com/juntossomosmais/atomium/pull/239/commits/8047697292e443a1506e4ea1fb0f1aea27e0412d